<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Canvas Eagles - Aircraft Maneuver Log</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,1,0" columnstretch="1,0">
    <item row="0" column="1">
     <widget class="QLineEdit" name="date">
      <property name="placeholderText">
       <string>Date</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QTabWidget" name="crew_tab">
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QGroupBox" name="turn_controls_grp">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Turn Controls</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,1,0,1">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Maneuver:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="maneuver_cmb"/>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Altitude:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QComboBox" name="alt_cmb">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="4">
        <widget class="QTabWidget" name="gun_tab"/>
       </item>
       <item row="2" column="0" colspan="4">
        <widget class="QPushButton" name="log_turn_btn">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>End Turn</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QTreeWidget" name="turn_log">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="animated">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>Maneuver / Crew Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Altitude / Action Taken</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Fuel Remaining</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="4" column="1">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="2">
       <widget class="PlanePartDamageTracker" name="fuselage_grp" native="true"/>
      </item>
      <item row="1" column="3">
       <widget class="PlanePartDamageTracker" name="tail_grp" native="true"/>
      </item>
      <item row="1" column="0">
       <widget class="PlanePartDamageTracker" name="engine_grp" native="true"/>
      </item>
      <item row="1" column="1">
       <widget class="PlanePartDamageTracker" name="wing_grp" native="true"/>
      </item>
     </layout>
    </item>
    <item row="0" column="0" rowspan="5">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="game_menu">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionCreate_plane"/>
    <addaction name="actionEdit_plane"/>
    <addaction name="actionLoad_Planes"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="select_plane_menu">
    <property name="title">
     <string>Select a Plane</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSelect_Scenario">
    <property name="title">
     <string>Select Scenario</string>
    </property>
    <addaction name="actionDogfight"/>
    <addaction name="actionBombing_Run"/>
    <addaction name="actionScouting_Mission"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionSet_Plane_Auto_Load_Location"/>
   </widget>
   <addaction name="game_menu"/>
   <addaction name="select_plane_menu"/>
   <addaction name="menuSelect_Scenario"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionLoad_Planes">
   <property name="text">
    <string>Load a Plane</string>
   </property>
  </action>
  <action name="actionCreate_plane">
   <property name="text">
    <string>Create a new plane</string>
   </property>
  </action>
  <action name="actionEdit_plane">
   <property name="text">
    <string>Edit an existing plane</string>
   </property>
  </action>
  <action name="actionDogfight">
   <property name="text">
    <string>Dogfight</string>
   </property>
  </action>
  <action name="actionBombing_Run">
   <property name="text">
    <string>Bombing Run</string>
   </property>
  </action>
  <action name="actionScouting_Mission">
   <property name="text">
    <string>Scouting Mission</string>
   </property>
  </action>
  <action name="actionSet_Plane_Auto_Load_Location">
   <property name="text">
    <string>Set Auto-Load Location</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlanePartDamageTracker</class>
   <extends>QWidget</extends>
   <header>turn_logger/PlanePartDamageTracker.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>549</x>
     <y>474</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
